modules/nw-dns-operator.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+// * networking/dns/dns-operator.adoc
+
+[id="dns-operator-{context}"]
+= DNS Operator
+
+The DNS Operator implements the `dns` API from the `operator.openshift.io` API
```

Comment:
Would it make sense to s/`dns`/`DNS`?

Reply:
The api kind is `DNS`, but the api is actually `dns` or `dnses` for plural.